hack/make-rules/test-cmd-util.sh

```diff
@@ -4236,8 +4289,8 @@ runTests() {
   pdb_max_unavailable=".spec.maxUnavailable"
   template_generation_field=".spec.templateGeneration"
   container_len="(len .spec.template.spec.containers)"
-  daemonset_image_field0="(index .spec.template.spec.containers 0).image"
-  daemonset_image_field1="(index .spec.template.spec.containers 1).image"
+  image_field0="(index .spec.template.spec.containers 0).image"
+  image_field1="(index .spec.template.spec.containers 1).image"
```

Comment:
We can replace `deployment_image_field` and `deployment_second_image_field` with these two as well